test/drivers/test_bqrrp.cc

```diff
@@ -154,12 +154,11 @@ TEST_F(TestBQRRP, BQRRP_full_rank_basic) {
     double d_factor = 1;//1.0;
     int64_t b_sz = 500;//500;
     double norm_A = 0;
-    double tol = std::pow(std::numeric_limits<double>::epsilon(), 0.85);
     auto state = RandBLAS::RNGState();
 
     BQRRPTestData<double> all_data(m, n, k);
-    RandLAPACK::BQRRP<double, r123::Philox4x32> BQRRP(true, tol, b_sz);
-    BQRRP.qr_tall = "cholqr";
+    RandLAPACK::BQRRP<double, r123::Philox4x32> BQRRP(true, b_sz);
+    BQRRP.qr_tall = RandLAPACK::BQRRP<double, r123::Philox4x32>::SubroutineType::qr_tall_subroutines::cholqr;
```

Comment:
At the top of this file just toss a``using RandLAPACK::BQRRP::Options`` (or ``using RandLAPACK::BQRRP::Subroutines``, depending on how you name things). Then lines like this can be less verbose.

Reply:
Tried that, result sin errors.

Reply:
Ah wait, works with a different syntax.

Reply:
kewl